shared/src/model/notifications.coffee

```diff
@@ -56,11 +56,10 @@ Notifications = {
       @emit('change')
 
   getActive: ->
-    return NOTICES[0] if NOTICES.length
+    active = clone(NOTICES)
```

Comment:
Hm I'm a bit confused by the interaction between `NOTICES`, `display` and `getActive`.

So `NOTICES` seems to contain an array of the things that are actually displayed and is set by the `display` method. `display` is called from `setCourseRole`, which means when you pick a dashboard?

`getActive` includes all the `NOTICES` but also adds extra notices from the pollers... (how are these different?)

I think we might want to document what's happening here for the future.